### PR TITLE
Diemo layer

### PR DESCRIPTION
**Pull request by procrastinatio**
[DIEMO](https://mf-chsdi3.int.bgdi.ch/shorten/7d7441fcaa)

styleUrl is: https://s3-eu-west-1.amazonaws.com/ltteo-diemo/ch.bfe.ladestellen-elektromobilitaet.styles.json

GeoJSON is https://s3-eu-west-1.amazonaws.com/ltteo-diemo/ch.bfe.ladestellen-elektromobilitaet.json

<jenkins>[Test link](https://mf-geoadmin3.int.bgdi.ch/teo_diemo/index.html)</jenkins>

### CODE REVIEW

**Comment by danduk82** [comment]
Update seems to work. But I see 2 bugs:
a) if we deactivate the layer it continues to reload the geojson
b) it seems not to take into account the HEAD request which returns 304 (see screenshot below)
see image

**Comment by gjn** [comment]
Regarding the reloading when not active, I think we need to simply add an additional filter in the realtime service here:

https://github.com/geoadmin/mf-geoadmin3/blob/master/src/components/map/RealtimeLayersService.js#L81

Add the existing filter `gaLayerFilters.selectedAndVisible`.

Not sure where the HEAD request is coming from, but accurate headers (max-age, must-revalidate) whould achieve probably the same with just one GET request. (To be tested, of course)

**Comment by davidoesch** [comment]
Take the pub transportation or meteoswiss data as reference

**Comment by procrastinatio** [comment]
1/ Only _ChargingStationId_ have coordinates, not the _EvseId_ . We already set faked coordinates a few meters apart.
2/ If you don't know what kind of plug an EveId has, it is useless.
3/ GeoJSON is cached, but as it is regenerated every minute from DynamoDB it has a different eTag every time (neither DynamoDB nor GeoJSON are ordered structure :-))

**Comment by gjn** [comment]
https://github.com/geoadmin/mf-geoadmin3/pull/4512 is ready for the issue described by Andrea (reloading layer even if not active)

File: src/components/map/LayersService.js [hunk 1]
```diff
@@ -200,6 +200,28 @@ goog.require('ga_urlutils_service');
...
+                    'ch.bfe.ladestellen-elektromobilitaet.json',
+                highlightable: true,
+                background: false,
+                updateDelay: 5000, // every 5sec!
+                topics: 'api,bfe,dev,ech,gewiss,inspire,kgs',
+                hasLegend: false,
+                type: 'geojson',
+                timeEnabled: false
+              };
+
```
Comment (gjn) [inline src/components/map/LayersService.js]: Needs to be removed before going to prod. Chsdi should integrate the layer.

File: src/components/StylesFromLiteralsService.js [hunk 2]
```diff
@@ -266,7 +265,8 @@ goog.provide('ga_stylesfromliterals_service');
         }
 
         if (!olStyles) {
-          return this.alertDebug_(value, feature.getId());
+          this.log_(value, feature.getId());
```
[the remark this answers is not in the document]
Comment (procrastinatio) [reply src/components/StylesFromLiteralsService.js]: Why? It's _really_ hard to debug style error with more than 1 style and two features.

**Pull request closed without merging**